app/views/soft-close/chosen.html

```diff
@@ -0,0 +1,22 @@
+{% extends "_wizard-form.html" %}
+{% set title = 'Have you agreed with a provider that you will start your NPQ in February?' %}
```

Comment:
```suggestion
{% set title = 'Have you agreed with a provider that you’ll start your NPQ in February?' %}
```
?

Reply:
Will do and made sure it's quotation marks ;)